NuGet.Config

```diff
@@ -7,7 +7,7 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
-    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption/nuget/v3/index.json" />
+    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
```

Comment:
Thanks for creating this PR. I thought local copy of the consumption feed is accessed first before reaching to upstream feeds.

Reply:
It works differently depending on the resource. 
For example, listing from flat container is different from downloading the nupkg.
| | list package versions | download |
|-|-|-|
| local view | Only ingested packages | Only ingested packages, matches the listed list |
| main view | All packages | Only ingested, but can download with auth, which we don't have |

Reply:
https://learn.microsoft.com/en-us/azure/devops/artifacts/how-to/public-feeds-upstream-sources?view=azure-devops&tabs=nuget#q-im-trying-to-restore-my-packages-but-i-keep-getting-a-401-unauthorized-error sort of talks about it, but not in detail.